dashboard/modules/reporter/reporter_agent.py

```diff
@@ -115,6 +115,12 @@ def jsonify_asdict(o) -> str:
     "node_mem_total": Gauge(
         "node_mem_total", "Total memory on a ray node", "bytes", ["ip", "SessionName"]
     ),
+    "node_mem_shared_bytes": Gauge(
```

Comment:
nit: why are we calling one "shared_bytes" while the other "shm_bytes"?

Reply:
Yeah, I guess my point was that given we had something called "node_mem_total", why don't we have consistent naming with "component_mem_xxx", e.g.
- node_mem_shared_bytes
- component_mem_shared_bytes
- node_mem_total_bytes (even)

Reply:
Hmm I made component_* consistent with other component metrics (component_rss_mb). And node consistent with other node metrics. `node_mem_total `. I agree it is better those two are also consistent, but I think it is probably too late... (due to backward compatibility)

Reply:
I at least used `component_shared_bytes`.

Reply:
hmm thinking more, it is already inconsistent because we use bytes instead of MB. So I will just follow your suggestion...